src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/ReclusterServiceIntTest.kt

```diff
@@ -35,31 +29,38 @@ class ReclusterServiceIntTest : MessagingMultiNodeTestBase() {
 
     @Test
     fun `should add a created record to a cluster if it is set to need attention`() {
-      val personA = createPerson(createRandomPersonDetails())
-      createPersonKey(status = UUIDStatusType.NEEDS_ATTENTION)
+      val personA = createPerson(createRandomProbationPersonDetails())
+      val cluster = createPersonKey(status = UUIDStatusType.NEEDS_ATTENTION)
         .addPerson(personA)
 
       stubPersonMatchUpsert()
       stubOnePersonMatchHighConfidenceMatch(matchedRecord = personA.matchId)
-      val cId = randomCId()
-      publishLibraMessage(libraHearing(firstName = randomName(), lastName = randomName(), cId = cId, dateOfBirth = randomDate().format(DateTimeFormatter.ofPattern("dd/MM/yyyy")), cro = "", pncNumber = randomPnc(), postcode = randomPostcode()))
+
+      val newPersonCrn = randomCrn()
+      probationEventAndResponseSetup(OFFENDER_ALIAS_CHANGED, ApiResponseSetup(crn = newPersonCrn))
 
       checkTelemetry(
         TelemetryEventType.CPR_RECORD_CREATED,
-        mapOf("C_ID" to cId),
+        mapOf("CRN" to newPersonCrn),
       )
 
-      awaitAssert { assertThat(personKeyRepository.findByPersonId(personA.personKey?.personId)?.personEntities?.size).isEqualTo(2) }
+      cluster.assertClusterIsOfSize(2)
+      cluster.assertClusterStatus(UUIDStatusType.NEEDS_ATTENTION)
     }
 
     @Test
-    fun `should do nothing when cluster already set as needs attention`() {
-      val personA = createPerson(createRandomPersonDetails())
+    fun `should update record and do nothing when cluster already set as needs attention`() {
+      val personA = createPerson(createRandomProbationPersonDetails())
       val cluster = createPersonKey(status = UUIDStatusType.NEEDS_ATTENTION)
         .addPerson(personA)
 
-      reclusterService.recluster(cluster, changedRecord = personA)
+      stubPersonMatchUpsert()
+      probationEventAndResponseSetup(OFFENDER_ALIAS_CHANGED, ApiResponseSetup(crn = personA.crn))
 
+      checkTelemetry(
```

Comment:
👍 although same remark on future obsolescence of OFFENDER_ALIAS_CHANGED as above

In fact we should probably have marked it as deprecated